pkg/inspect/inspect.go

```diff
@@ -14,14 +15,25 @@ const (
 	REALITY_SOURCE
 )
 
+type LaunchableVersion struct {
+	Location string                    `json:"location,omitempty"`
```

Comment:
typically `LaunchableStanza` would be the structure you'd see this presented in, but that's more information than `p2-inspect` should provide. You're right that this struct might be useful elsewhere, but I don't think it is today